src/meteodatalab/operators/atmo.py

```diff
@@ -24,6 +24,54 @@ def pv_sw(t):
     return pc.b1 * np.exp(pc.b2w * (t - pc.b3) / (t - pc.b4w))
 
 
+def pv_si(t):
+    """Pressure of water vapor at equilibrium over ice.
+
+    Parameters
+    ----------
+    t : xr.DataArray
+        temperature (in Kelvin)
+
+    Returns
+    -------
+    xr.DataArray
+        pressure of water vapor in Pascal
+
+    """
+    return pc.b1 * np.exp(pc.b2i * (t - pc.b3) / (t - pc.b4i))
+
+
+def pv_sm(t):
+    """Pressure of water vapor at equilibrium over mixed phase.
+
+    Parameters
+    ----------
+    t : xr.DataArray
+        temperature (in Kelvin)
+
+    Returns
+    -------
+    xr.DataArray
+        pressure of water vapor in Pascal
+
+    """
+    tice_only = pc.b3 - 23
+    dtice = pc.b3 - tice_only
+
+    if t > pc.b3:
```

Comment:
This needs to be implemented using the `where` method as `t` is a field and the condition might not apply uniformly